processing_services/example/api/global_moth_classifier.py

```
@@ -0,0 +1,249 @@
+        Args:
+            detections: List of Detection objects with cropped images
+
+        Returns:
+            List of Detection objects with added classifications
+        """
+        if not detections:
+            return []
+
+        # Ensure model is compiled
+        if self.model is None:
+            raise RuntimeError("Model not compiled. Call compile() first.")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)